Source/IGListSectionController.m

```diff
@@ -89,5 +89,17 @@ - (void)didDeselectItemAtIndex:(NSInteger)index {}
 - (void)didHighlightItemAtIndex:(NSInteger)index {}
 
 - (void)didUnhighlightItemAtIndex:(NSInteger)index {}
+    
+- (BOOL)canMoveItemAtIndex:(NSInteger)index {
+    return NO;
+}
+
+- (BOOL)canMoveItemAtIndex:(NSInteger)sourceItemIndex toIndex:(NSInteger)destinationItemIndex {
```

Comment:
Could this be exposed for overriding in order to allow users to prevent moving to specific indices? 
Currently even if `canMoveItemAtIndex:` returns false for a specific index that item can still be moved when you move another item into its place.